src/response_to_json_translator.cc

```diff
@@ -125,11 +129,23 @@ bool ResponseToJsonTranslator::TranslateMessage(
 
   // Do the actual translation.
   status_ = ::google::protobuf::util::BinaryToJsonStream(
-      type_resolver_, type_url_, proto_in, &json_stream, json_print_options_);
+      type_resolver_, type_url_, proto_in, &json_stream,
+      options_.json_print_options);
+
   if (!status_.ok()) {
     return false;
   }
 
+  // Append a newline delimiter after the message if needed.
+  if (streaming_ && options_.stream_newline_delimited) {
```

Comment:
IIUC, the last message in the stream will also have a newline. Is this intended? Which of the following is the correct behavior for a delimiter according to gRPC gateway?

1) `A\nB\nC\n`
2) `A\nB\nC`

Reply:
Based on https://github.com/ndjson/ndjson-spec I think the current implementation is correct.

> Each JSON text MUST conform to the [[RFC7159](http://www.ietf.org/rfc/rfc7159.txt)] standard and MUST be written to the stream followed by the newline character \n (0x0A).

LGTM, thanks